_rules/document-title-is-descriptive.md

```diff
@@ -0,0 +1,200 @@
+---
+name: Document title is descriptive
+rule_type: atomic
+description: |
+  This rule checks that the document title describes the topic or purpose of the document.
+
+success_criterion: 
+- 2.4.2 # Page Titled (A)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Anne Thyme Nørregaard
+- Corbb O’Connor 
```

Comment:
This person is not in contributors list. This has to be updated.

Reply:
Fixed.

Reply:
@jkodu, thanks for pointing this out. I have added @corbb to the Auto-WCAG contributor's list now :)